drivers/SmartThings/zigbee-button/src/aqara/init.lua

```diff
@@ -79,9 +81,15 @@ local function device_init(driver, device)
     end
 end
 
-  local function added_handler(self, device)
-    device:send(cluster_base.write_manufacturer_specific_attribute(device,
-                PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1))
+local function added_handler(self, device)
+    if device:get_model() == "lumi.remote.b1acn02" then
+      device:send(cluster_base.write_manufacturer_specific_attribute(device,
+      PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID_T1, MFG_CODE, data_types.Uint8, 1))
+    elseif device:get_model() == "lumi.remote.acn003" then
+      device:send(cluster_base.write_manufacturer_specific_attribute(device,
+      PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID_E1, MFG_CODE, data_types.Uint8, 2))
+    end
```

Comment:
a comment indicating what these both do would be helpful

Reply:
ok,when the wireless switch T1 accesses the network, the gateway sends private attribute 0009 to make the device no longer distinguish between the standard gateway and the aqara gateway. When wireless switch E1 is connected to the network, the gateway delivers private attribute 0125 to enable the device to send double-click and long-press packets. Wireless switch T1 supports double and long press by default, while wireless switch E1 does not distinguish between standard gateway and aqara gateway by default. This is the point where they issue different instructions when they enter the network.